akka-stream/src/main/scala/akka/stream/impl/Timers.scala

```diff
@@ -275,11 +275,15 @@ import akka.stream.stage._
           val now = System.nanoTime()
           // Timer is reliably cancelled if a regular element arrives first. Scheduler rather schedules too late
```

Comment:
Any suggestion?

Reply:
if i understand correctly:

`// reschedule for negative time drift to ensure not triggered too early

Reply:
we could also remove the entire comment and adapt log.warning to something like this:
```
log.warning(
              s"Timer should have triggered only after deadline but now is $now and deadline was $nextDeadline diff $diff. (time running backwards?) Reschedule instead of emitting.")
```

This imho would be clearer to read the code and for the one stumbling upon this log...